projectsmember/Pages/CooperationRequest/Profiles/HamidForoughnia.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> رزومه</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    
+    <style>
+        .material-symbols-outlined {
+            font-variation-settings:
+                    'FILL' 0,
+                    'wght' 400,
+                    'GRAD' 0,
+                    'opsz' 48
+        }
+    </style>
+
+</head>
+<body  dir="rtl">
+<div class="container border">
+    <div class="row">
+        <h1 class="mt-4">حمید فروغ نیا</h1>
+        <a href="https://dotnet.microsoft.com/en-us/languages/csharp">
```

Comment:
Links to other websites... those are like the credits we give them... unless we use rel="nofollow" so the search engine crawlers do not count them as our out-links. 
that overall affects our website's ranking over time.